server/integration/jgroups/src/main/resources/subsystem-templates/cloud-jgroups.xml

```diff
@@ -141,6 +141,26 @@
                 <protocol type="MFC" />
                 <protocol type="FRAG3" />
             </stack>
+            <stack name="dns-ping">
+                <transport type="TCP" socket-binding="jgroups-tcp">
+                    <property name="logical_addr_cache_expiration">360000</property>
+                </transport>
+                <protocol type="dns.DNS_PING">
+                    <property name="dns_query">${jgroups.dns_ping.dns_query:example-infinispan-ping.default.svc.cluster.local}</property>
```

Comment:
I would not give this a default value, I'd just define it as: `${jgroups.dns_ping.dns_query:}`, because there's no valid value as default. When deploying it in cloud the template or operator would provide the right value based on the environment.

Reply:
ok, done